.gitignore

```diff
@@ -5,7 +5,8 @@ node_modules/
 packages/
 workbooks-proprietary/
 
-Build/Local.props
```

Comment:
wasn't local.props where you can store info about your preferred Mac build server?

Reply:
Yes. But it doesn't exist anymore.

Reply:
Did we want to bring it back in the new build world? I can file an issue if you want. The README still has info about it.